api/envoy/config/filter/http/ext_authz/v2alpha/ext_authz.proto

```diff
@@ -62,4 +62,10 @@ message HttpService {
   // Sets a list of headers that should be not be sent *from the authorization server* to the
   // upstream.
   repeated string response_headers_to_remove = 3;
```

Comment:
Shouldn't this also become a whitelist? Aren't we going to have the same problem? IMO it's a lot more clear if we are consistent here and make both a whitelist. Should we switch this? I'm not sure we need to go through the full deprecation cycle for this, but up to you.

Reply:
Let me take a look into this. I remember that there was a bunch of discussions around it, but I agree that whitelist makes much more sense here.